salt/returners/mongo_future_return.py

```diff
@@ -17,6 +18,12 @@
     mongo.password: <MongoDB user password>
     mongo.port: 27017
 
+Or single URI:
+
+.. code-block:: yaml
+
+   mongo.uri: URI
```

Comment:
Please provide an example of the uri format.

Thanks,
Daniel

Reply:
Done

Reply:
I have updated doc, and changed LooseVersion in mongo_future_return and mongo_return. I did some more extra handling so that only uri or host can be specified in the config file.

Reply:
perfect, thanks!